tests/misc_env/install_prereq.py

```diff
@@ -381,17 +381,20 @@ def registry_login(ceph, distro_ver):
         }
     ]
 
-    if (
-        config.get("registry_credentials")
-        and config["registry_credentials"]["registry"] != "registry.redhat.io"
-    ):
-        registries.append(
-            {
-                "registry": config["registry_credentials"]["registry"],
-                "user": config["registry_credentials"]["username"],
-                "passwd": config["registry_credentials"]["password"],
-            }
+    if config.get("registry_credentials"):
```

Comment:
- same as above. 
- Would suggest to pass registry name to function, expect creds in return.

Reply:
Here the plan was to add all the registries to .docker/config.json which are present in .cephci.yaml
Which will not harm anywhere in the execution.